### ISSUE TITLE
Default Filter(s)

### ISSUE THREAD

**Issue by siripravi** [luyadev/luya-module-admin#502]
### What steps will reproduce the problem?
I have defined these filters in my NgModel:
`public function ngRestFilters() {
        return [
            'Deleted' => self::find()->where(['=', 'is_deleted', 1]),
            'Active' => self::find()->where(['=', 'is_deleted', 0]),
        ];
    }`

### What is the expected result?
I want to show 'Active' Items by default.
Is it possible to set some filter as default?

### Additional infos

| Q                | A
| ---------------- | ---
| LUYA Version     | 
| PHP Version      |
| Platform         | Apache/XAMPP/MAMPP/etc.
| Operating system | Windows/Linux Server/OSX/etc.

**Comment by nadar** [comment]
Its not possible, but if you have selected a filter, it will be stored in the user settings and will be the default filter for your account. does that help?

**Comment by siripravi** [comment]
Sure. It would be helpful. Is it automatic? How to do it? Any documentation on this?

**Comment by nadar** [comment]
Its not available yet, but we can add this feature :+1: should not be to hard to develop.

**Comment by siripravi** [comment]
Its not filter on just one single field. We should be able to have something called a 'Filter Group', that should hold a set of filters(with search criteria) on several fields.
The user should be able to use(apply) one or more of these filter groups to the list view. Possibly we can show these filter groups as a dropdown list or a button group etc.

**Comment by nadar** [comment]
but this is not the topic you mentioned in the start of the issue?

**Comment by siripravi** [comment]
Yes. What I mentioned was an enhancement to the previous requirement.  I first talked about default filter criteria on one field only.

**Comment by nadar** [comment]
Well, filters can have as much criterias as you like, its a where condition. But maybe you like to use Tags, then you can also filter by tags.

**Comment by siripravi** [comment]
Exactly. I dont know how feasible this requirement is, but would be helpful.

**Comment by siripravi** [comment]
Sure.

**Comment by nadar** [comment]
@siripravi You can already use tags  to filter data, just add the TaggableTrait trait and TaggableActiveWindow and you are done.
